packages/fast-usdc/test/exos/advancer.test.ts

```diff
@@ -399,7 +399,7 @@ test('will not advance same txHash:chainId evidence twice', async t => {
 
   t.deepEqual(inspectLogs(1), [
     'Advancer error:',
-    '"[Error: Transaction already seen: \\"seenTx:[\\\\\\"0xc81bc6105b60a234c7c50ac17816ebcd5561d366df8bf3be59ff387552761702\\\\\\",1]\\"]"',
+    '"[Error: Transaction already seen: \\"seenTx:[\\\\\\"0xc81bc6105b60a234c7c50ac17816ebcd5561d366df8bf3be59ff387552761702\\\\\\"]\\"]"',
```

Comment:
In my book, if you can see a difference from a test using an external API, it's not a refactor.
But we don't seem to have a shared definition across the team. Oh well.

Reply:
I chose `refactor` since this part of the implementation is invisible to contract consumers and other exos in the contract. I can amend both of the refactors to `feat`, though.

Reply:
ah. so it's an internal API. fair enough.